_ont/ont-zte-f6005.md

```diff
@@ -170,4 +171,8 @@ After these steps, power-cycle ONT and login via TELNET with `root\admin` creden
 
 # Known Bugs
 
-In versions V6.0.10N14 and V6.0.10P2N02 buffer size is suboptimal: because of this the ONT is unable to work at full speed during uploads if the server is geographically, and/or latency-wise, far. There are no known problems if there is only one person on the GPON tree.
+The buffer size of the hardware is suboptimal is suboptimal: because of this the ONT is unable to work at full speed during uploads if the server is geographically, and/or latency-wise, far. There are no known problems if there is only one person on the GPON tree. 
```

Comment:
```suggestion
The buffer size of the hardware is suboptimal: because of this the ONT is unable to work at full speed during uploads if the server is geographically and/or latency-wise far. There are no known problems if there is only one subscriber on the GPON tree. 
```